package.json

```diff
@@ -114,6 +133,10 @@
     "babel-relay-plugin": "^0.9.0"
   },
   "lint-staged": {
-    "*.@(ts|tsx)": ["tslint --fix", "npm run prettier-write --", "git add"]
+    "*.@(ts|tsx)": [
+      "tslint --fix",
+      "npm run prettier-write --",
+      "git add"
+    ]
   }
 }
```

Comment:
I've been avoiding adding package.json and yarn.lock for this reason. It's been out of sync for ages, and not sure what to do :/